src/Iced/Internal.hs

```diff
@@ -2,8 +2,8 @@ module Iced.Internal where
 
 import Foreign
 
-foreign export ccall "free_stable_ptr"
+foreign export capi "free_stable_ptr"
```

Comment:
leave it as `ccall` for now, as I mentioned in another comment we can discuss `capi` separately